<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <meta name="author" content="David Leon (Dawei Leng)">
  
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>COVID-19 Worldwide Publications - Targeting COVID-19 Portal</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700" />

  <link rel="stylesheet" href="../css/theme.css" />
  <link rel="stylesheet" href="../css/theme_extra.css" />
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" />
  
  <script>
    // Current page data
    var mkdocs_page_name = "COVID-19 Worldwide Publications";
    var mkdocs_page_input_path = "research_progress.md";
    var mkdocs_page_url = null;
  </script>
  
  <script src="../js/jquery-2.1.1.min.js" defer></script>
  <script src="../js/modernizr-2.8.3.min.js" defer></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
  
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> Targeting COVID-19 Portal</a>
        
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="..">Home</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../clinical/">COVID-19 Worldwide Clinical Trials</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../preclinical/">COVID-19 Worldwide Preclinical Studies</a>
                    </li>
                </ul>
                <ul class="current">
                    <li class="toctree-l1 current"><a class="reference internal current" href="./">COVID-19 Worldwide Publications</a>
    <ul class="current">
    </ul>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../Data_analytics/">COVID-19 GHDDI Data Analytics</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../nCov_Structures/">COVID-19 GHDDI Structure modeling</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../computational/">COVID-19 GHDDI Computational Studies</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../ghddi_news/">COVID-19 GHDDI News</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../contact_us/">COVID-19 Discussion</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../useful_links/">COVID-19 Useful Links</a>
                    </li>
                </ul>
                <p class="caption"><span class="caption-text">Literature Review and Data Download</span></p>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../CoV_Experiment_Data/">Coronavirus Data Download</a>
                    </li>
                    <li class="toctree-l1"><a class="reference internal" href="../nCov_Targets/">Coronavirus Potential Therapeutics Review</a>
                    </li>
                    <li class="toctree-l1"><a class="reference internal" href="../review_paper/">Coronavirus Review Articles</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../history/">Github History</a>
                    </li>
                </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">Targeting COVID-19 Portal</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>COVID-19 Worldwide Publications</li>
    <li class="wy-breadcrumbs-aside">
      
    </li>
  </ul>
  
  <hr/>
</div>
          <div role="main">
            <div class="section">
              
                <h2 id="covid-19-publication-feeds">COVID-19 publication feeds</h2>
<ul>
<li><a href="https://dimensions.ai">dimensions.ai</a> currently provides a list of research articles and other research items updated daily regarding COVID-19 and coronavirus with <a href="https://covid-19.dimensions.ai/">this search query</a>.</li>
</ul>
<h2 id="nlp-topic-model">NLP Topic Model</h2>
<p>The table below shows recommended papers from an optimized topic model using literature data aggregated from different sources such as BioRxiv and PubMed. Some of the topics sorted in this optimized model did not meet the filter requirements and were removed. There are a total of 20 topics shown in this tabulated table, and the top ten papers are displayed in each topic tab. This topic model results will be updated daily, and the model itself will be updated monthly. Our NLP topic model is not guaranteed to be 100% accurate for each topic's recommendations, so this table should be used as a reference for further reading.</p>
<iframe src="https://mg.k-global-cdn.ml/ldatopics.html" style="border:0px #ffffff none;" name="preclinical" frameborder="1" marginheight="0px" marginwidth="0px" height="500px" width="700px" scrolling="yes" allowfullscreen></iframe>

<hr />
<h2 id="selected-publications">Selected publications</h2>
<p>Last update: March 23, 2020</p>
<h2 id="2020">2020</h2>
<p>2020, Mar 23</p>
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.03.22.002386v1">A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug-Repurposing</a><ul>
<li>An outbreak of the novel coronavirus SARS-CoV-2, the causative agent of COVID-19 respiratory disease, has infected over 290,000 people since the end of 2019, killed over 12,000, and caused worldwide social and economic disruption. There are currently no antiviral drugs with proven efficacy nor are there vaccines for its prevention. Unfortunately, the scientific community has little knowledge of the molecular details of SARS-CoV-2 infection. To illuminate this, we cloned, tagged and expressed 26 of the 29 viral proteins in human cells and identified the human proteins physically associated with each using affinity-purification mass spectrometry (AP-MS), which identified 332 high confidence SARS-CoV-2-human protein-protein interactions (PPIs). Among these, we identify 66 druggable human proteins or host factors targeted by 69 existing FDA-approved drugs, drugs in clinical trials and/or preclinical compounds, that we are currently evaluating for efficacy in live SARS-CoV-2 infection assays. The identification of host dependency factors mediating virus infection may provide key insights into effective molecular targets for developing broadly acting antiviral therapeutics against SARS-CoV-2 and other deadly coronavirus strains.</li>
</ul>
</li>
</ul>
<p>2020, Mar 14</p>
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.03.14.988345v1">SARS-CoV-2 invades host cells via a novel route: CD147-spike protein</a><ul>
<li>The discovery of the new route CD147-SP for SARS-CoV-2 invading host cells provides a critical target for development of specific antiviral drugs. This study publishes a research finding that SARS-CoV-2 invaded host cells via a novel route of CD147-spike protein (SP), mediating the viral invasion with several in vitro assays validating this result. </li>
</ul>
</li>
</ul>
<hr />
<p>2020, Mar 12</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_986836v1.pdf">A data-driven drug repositioning framework discovered a potential therapeutic agent targeting COVID-19 </a><ul>
<li>A data-driven framework that combines both machine learning and statistical analysis methods was used to integrate abailable coronavirus data and identify drug candidates against the novel coronavirus. Through this screening, one inhibitor, CVL208, may be repurposed to treat COVID-19 with in vitro assays also showing effective inhibitory activity against the novel coronavirus.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.03.09.983247v1">Inhibition of SARS-CoV-2 infection (previously 2019-nCoV) by a highly potent pan-coronavirus fusion inhibitor targeting its spike protein that harbors a high capacity to mediate membrane fusion</a><ul>
<li>A SARS-CoV-2 spike (S) protein-mediated cell-cell fusion assay was used, and it was found that SARS-CoV-2 showed plasma membrane fusion capacity superior to that of SARS-CoV. A series of lipopeptides was generated and found that the EK1C4 was the most potent fusion inhibitor against SARS-CoV-2 S protein-mediated membrane fusion and pseudovirus infection with IC50s of 1.3 and 15.8 nM, about 241- and 149-fold more potent than that of EK1 peptide. Intranasal application of EK1C4 before or after challenge with HCoV-OC43 protected mice from infection, suggesting that EK1C4 could be used for prevention and treatment of infection.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Mar 10</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_smprocvdi.pdf">Structure of Mpro from COVID-19 virus and discovery of its inhibitors</a><ul>
<li>This study combined structure-assisted drug design, virtual drug screening, and high-throughput screening to identify new drug leads that target the COVID-19 virus main protease (Mpro). Six compounds were identified which inhibit Mpro with IC50 values ranging from 0.67 to 21.4 μM; one of which, ebselen, also exhibited strong antiviral activity in cell-based assays.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Mar 8</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_aadntciasc2.pdf">AI-aided design of novel targeted covalent inhibitors against SARS-CoV-2</a><ul>
<li>A novel deep Q-learning network with the fragment-based drug design (ADQN-FBDD) was developed for generating potential lead compounds targeting SARS-CoV-2 3CLpro using a recently published structure (PDB IDL: 6LU7). 47 lead compounds were published from this AI model and are intended to be used for further drug development research.</li>
<li>Supplementary data can be found at https://github.com/tbwxmu/2019-nCov</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Mar 3</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_icsc2fuscp.pdf">Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient </a><ul>
<li>This virus was extracted from the first U.S. patient, and this sample's viral sequence, replication properties, and cell culture tropism were characterized. The virus isolate available to the public health community by depositing into two virus reagent repositories: BEI resources and The World Reference Center for Emergine Viruses and Arboviruses.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 29</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_tedc19pcd.pdf">"Therapeutic effects of dipyridamole on COVID-19 patients with coagulation dysfunction"</a><ul>
<li>Details about the effects of dipyridamole on COVID-19 patients. It was reported that Two weeks after initiation of DIP treatment, 3 of the 6 severe cases (60%) and all 4 of the mild cases (100%) were discharged from the hospital. This article references a related clinical trial ChiCTR2000030055.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_nwaa036.pdf">On the origin and continuing evolution of SARS-CoV-2</a><ul>
<li>Through population genetic analysis, this virus is reported to have evolved into two major types (L and S) that are well defined by two different SNPs. The L type (~70%) is currently more prevalent than the S type (~30%), but the S type was found to be the ancestral version but less aggresssive. </li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 28</p>
<ul>
<li><a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa145/5766118">Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</a><ul>
<li>Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format.</li>
<li>Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for 2019-nCoV. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines.</li>
<li>Preprint released on February 2, 2020. Published on February 28, 2020 </li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 27</p>
<ul>
<li><a href="https://www.nature.com/articles/d41587-020-00003-1">"Coronavirus puts drug repurposing on the fast track"</a><ul>
<li>Existing antivirals and knowledge gained from the SARS and MERS outbreaks gain traction as the fastest route to fight the current coronavirus epidemic. This news briefly reviewed the repurposed agents targeting SARS-CoV2, and pointed out that although virtual screens can potentially accelerate drug discovery, the predicted compounds still need to be tested carefully with experiments.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 25</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_faceruscolb.pdf">"Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses"</a><ul>
<li>This article shares a recent development where researchers developed an approach to rapidly screen lineage B betacoronaviruses, and their ability to infect cell types from different species. They found that bypassing the host protease processing barrier allows several lineage B viruses to enter human cells. These different lineage B viruses can also recombine to gain entry into human cells with human ACE2 being the receptor for 2019-nCoV.</li>
<li>Preprint released on January 22, 2020. Published on February 25, 2020 </li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 24</p>
<ul>
<li><a href="https://www.nature.com/articles/s41421-020-0147-1">Comparative genetic analysis of the novel coronavirus (2019-nCoV/SARS-CoV-2) receptor ACE2 in different populations</a><ul>
<li>This Nature Letter to the Editor article describes a study of the genetic analysis for novel coronavirus. They systematically analyzed coding-region variants in ACE2 and the eQTL variants to compare the genomic characteristics of ACE2 among different populations. It was found that no direct evidence was identified genetically supporting the existence of coronavirus S-protein binding-resistant ACE2 mutants in different populations.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 21</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_mcmrb2nvilchc.pdf">"A Multiscale and Comparative Model for Receptor Binding of 2019 Novel Coronavirus and the Implication of its Life Cycle in Host Cells"</a><ul>
<li>A computational study with results indicating that the association of the new virus to the receptor is slower than SARS. This difference was integrated into a rigid-body mathematical model and found that this can result in a longer incubation period while maintaining a relatively higher level of viral concentration in the human body.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 18</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_sodflhace2.pdf">"Structure of dimeric full-length human ACE2 in complex with B0AT1"</a><ul>
<li>2.9 Å  resolution  cryo-EM  structure  of  full-length  human  ACE2  in  complex  with B0AT1 is presented in this study. The complex exhibits open and closed conformations due to the shifts of the peptidase domains of ACE2. Structural modelling suggests that this complex can bind two S proteins simultaneously. </li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 16</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_cems2nspc.pdf">"Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation"</a><ul>
<li>This study produced 3.5 A-resolution structures of the 2019-nCoV S trimer in the prefusion conformation. Additionally, results from this study indicate that this S protein binds ACE2 with higher affinity than the previous SARS-CoV S protein through biophysical and structural evidence. </li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 14</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/ASystemsApproachToInfectiousDisease.pdf">"A systems approach to infectious disease"</a><ul>
<li>A framework for a systems biology approach to infectious disease in three parts: discovery — the design, collection and analysis of omics data; representation — the iterative modelling, integration and visualization of complex data sets; and application — the interpretation and hypothesis-based inquiry towards translational outcomes.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 13</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_phpd2c.pdf">"Potentially highly potent drugs for 2019-nCoV"</a><ul>
<li>Drug repurposing was used in this study to identify many FDA approved drugs are potentially active against 2019-nCoV. This was done using a SARS 3CL protease X-ray crystal structure to create a 3D homology structure. Additionally, protein-ligand complexes were used as training data for a deep learning model to predict binding affinities; 1465 FDA-approved drugs were tested and the predicted binding affinities were reported in this study's results section.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 11</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_efwncocebvd.pdf">"The  Essential  Facts  of  Wuhan  Novel  Coronavirus  Outbreak  in  China  and  Epitope-based Vaccine Designing against 2019-nCoV"</a><ul>
<li>This article describes a study to design possible epitope-based subunit vaccines against the 2019-nCoV using a variety of strategies. Many amino acid sequences were published in this study, including three vaccine construct sequences. </li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01828?from=timeline&amp;isappinstalled=0">α-Ketoamides as Broad-Spectrum Inhibitors of Coronavirus and Enterovirus Replication: Structure-Based Design, Synthesis, and Activity Assessment</a><ul>
<li>The main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-site architecture and a unique requirement for glutamine in the P1 position of the substrate. Six crystal structures of protease–inhibitor complexes were determined as part of this study. Compounds synthesized were tested against the recombinant proteases as well as in viral replicons and virus-infected cell cultures; most of them were not cell-toxic. </li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 10</p>
<ul>
<li><a href="https://www.nature.com/articles/d41573-020-00016-0">"Therapeutic options for the 2019 novel coronavirus (2019-nCoV)"</a><ul>
<li>Therapeutic options in response to the 2019-nCoV outbreak are urgently needed. Here, we discuss the potential for repurposing existing antiviral agents to treat 2019-nCoV, some of which are already moving into clinical trials.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 7</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_p23clpidgdl.pdf">"Potential 2019-nCoV 3C-like protease inhibitors designed using generative deep learning approaches"</a><ul>
<li>This article from Insilico Medicine utilizes its generative chemistry pipeline to design novel molecules for 2019-nCoV through several generative chemistry approaches. Inputs for this model include 2019-nCoV 3C-like protease crystal structure, homology model, co-crystalized fragment, and a protease datasets. Several structures were generated and shown in the paper's results section. The most recent data results are on their official website: insilico.com/ncov-sprint.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 6</p>
<ul>
<li><a href="http://rs.yiigle.com/yufabiao/1179575.htm">"2019-nCoV: new challenges from coronavirus"</a><ul>
<li>According to current observations, 2019-nCoV is weaker than SARS in pathogenesis, but has stronger transmission competence; it's mechanism of cross-species spread might be related with angiotensin-converting enzyme Ⅱ (ACE2), which is consistent with the receptor SARS-CoV.</li>
<li>The emergence of 2019-nCoV reminds us once again of the importance of establishing a systematic coronavirus surveillance network. It also poses new challenges to prevention and control of the emerging epidemic and rapidly responses on scientific research.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 5</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_nbdrhc.pdf">"Network-based Drug Repurposing for Human Coronavirus"</a><ul>
<li>The researchers present an integrative, antiviral drug repurposing methodology implementing a systems pharmacology-based network medicine platform, quantifying the interplay between the HCoV-host interactome and drug targets in the human protein-protein interaction network.</li>
<li>Using network proximity analyses of drug targets and known HCoV-host interactions in the human protein-protein interactome, the researchers computationally identified 135 putative repurposable drugs for the potential prevention and treatment of HCoVs, and prioritized 16 potential anti-HCoV repurposable drugs (including <strong>melatonin</strong>, <strong>mercaptopurine</strong>, and <strong>sirolimus</strong>) that were further validated by enrichment analyses of drug-gene signatures and HCoV-induced transcriptomics data in human cell lines.</li>
<li>Three potential drug combinations (including sirolimus plus dactinomycin, mercaptopurine plus melatonin, and toremifene plus emodin) captured by the Complementary Exposure pattern.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.02.02.931162v2">"Genomic variance of the 2019-nCoV coronavirus"</a><ul>
<li>A phylogenetic tree was constructed of novel coronavirus as well as other coronaviridae such as bat coronavirus and SARS. The sequence similarity of &gt;99% was confirmed between all 2019-nCoV genomes, and the closest bat coronavirus genome shared a 96.2% sequence identity. A proteomic comparison with other coronaviridae was also performed and key aminoacidic differences were identified for potential antiviral strategies.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_lftppuptosar.pdf">"Learning from the Past: Possible Urgent Prevention and Treatment Options for Severe Acute
Respiratory Infections Caused by 2019-nCoV"</a><ul>
<li>The spike protein used for host cell infection although shows mutations but remains a viable target for development of biologics. Other key drug targets, including RdRp and 3CLpro, share a strikingly high (&gt;95%) homology to SARS‐CoV. This paper suggests 4 potential drug candidates: an ACE2‐based peptide, remdesivir, 3CLpro-1, and novel sulfone protease inhibitor to possibly treat patients.</li>
<li>Preprint released on January 27, 2020. Published on February 5, 2020 </li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 4</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_phpd2c.pdf">"Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro"</a><ul>
<li>This Letter to the Editor article first identifies several existing antiviral drugs to treat related viral infections. Through standard in vitro assays, the researchers found that remdesivir and chloroquine are highly effective in the control of 2019-nCoV infection in vitro, and recommend further human patient assessment based on these drugs' previous safety track record on human patients.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.02.02.931162v2">"Machine learning-based analysis of genomes suggests associations between Wuhan 2019-nCoV and bat Betacoronaviruses"</a><ul>
<li>Machine learned models and digital signal processing were used to classify 2019-nCoV genomes. After mapping using a two-dimensional numerical representation, the magnitude spectra was computed using a Discrete Fourier Transform. Afterwards, a distance matrix was calculated using Pearson Correlation Coefficients and were used as features for different machine learning algorithms. Through 10-fold cross validation, the classification accuracy for coronavirus was over 90%, indicating that 2019-nCoV can be classified closest to sarbecovirus within betacoronavirus.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_bpt2ncard.pdf">"Baricitinib as potential treatment for 2019-nCoV acute respiratory disease"</a><ul>
<li>Using BenevolentAI knowledge graph to search for approved drugs that could help 2019-nCoV, baricitinib is predicted to reduce the ability of the virus to infect lung cells. This study focuses on the disruption of AAK1 which might interrupt the passage of the virus into cells and also the intracellular assembly of virus particles. One of the six high-affinity AAK1-binding drugs was baricitinib, and this may have potential to reduce both the viral entry and the inflammation in patients.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.01.30.927889v1">Machine intelligence design of 2019-nCoV drugs</a><ul>
<li>A family of potential 2019-nCoV drugs generated by a machine intelligence-based generative network complex (GNC) is reported in this study. A SARS-CoV protease inhibitor dataset from ChEMBL, and a binding affinity training set comtaining PDBbind data were used for this model. The model prediccts 8000 potential drugs evaluated based on a latent space binding predictor and a 2D fingerprint predictor. Drug candidates are further evaluated by deep learning models, and logP, logS, and SA score were computed. These results indicate that anti-2019-nCoV compounds built in this study are in the top 15 and are predicted to work better than anti-HIV drugs current used like lopinavir and ritonavir.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.02.03.933226v1">"Preliminary identification of potential vaccine targets for 2019-nCoV based on SARS-CoV immunological studies"</a><ul>
<li>This study attempts to gain insight into vaccine design against 2019-nCoV. There is high genetic similarity between this virus and SARS coronavirus. Therefore, by screening SARS-CoV-derived B cell and T cell epitopes, a set of B cell and T cell epitopes derived from spike and nucleocapsid proteins mapped identically to 2019-nCoV proteins. There were no mutations observed meaning immune targeting of these epitopes may offer protection against 2019-nCoV.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 3</p>
<ul>
<li>Gilead Sciences and 2019 Novel Coronavirus (2019-nCoV)<ul>
<li>Remdesivir from Gilead Sciences <a href="https://www.bioworld.com/articles/432804-gileads-remdesivir-enters-china-phase-iii-trial-to-fight-coronavirus">enters Phase III</a> clinical trials in China on February 3, 2020.  </li>
<li>An official <a href="https://www.gilead.com/news-and-press/company-statements/gilead-sciences-statement-on-the-company-ongoing-response-to-the-2019-new-coronavirus">news release</a> from Gilead expressing their committment to respond to this outbreak by testing their novel investigational compound remdesivir in a controlled clinical trial as a treatment for 2019-nCoV.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 2</p>
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.01.31.928796v1">Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</a><ul>
<li>Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format.</li>
<li>Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for 2019-nCoV. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_hipwncdla.pdf">"Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm"</a><ul>
<li>Using deep learning algorithms, this article builds a virus host prediction model in order to predict the potential virus hosts. The built model suggests that 2019-nCoV has close infectivity with other human coronaviruses, especially SARS and MERS. Additionally, by comparing predicted infectivity patterns, bat coronaviruses are  assigned  with  more  similar  infectivity  patterns  with  2019-nCoVs. It was also suggested that mink viruses show a closer infectivity pattern to 2019-nCov showing that bat and mink may be two candidate reservoirs  of  2019-nCov.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.01.31.929547v1">Predicting commercially available antiviral drugs that may act on the novel coronavirus (2019-nCoV), Wuhan, China through a drug-target interaction deep learning model</a><ul>
<li>We used our pre-trained deep learning-based drug-target interaction model called Molecule Transformer-Drug Target Interaction (MT-DTI) to identify commercially available drugs that could act on viral proteins of 2019-nCoV.</li>
<li>The result showed that atazanavir, an antiretroviral medication used to treat and prevent the human immunodeficiency virus (HIV), is the best chemical compound, showing a inhibitory potency with Kd of 94.94 nM against the 2019-nCoV 3C-like proteinase, followed by efavirenz (199.17 nM), ritonavir (204.05 nM), and dolutegravir (336.91 nM). </li>
<li>Lopinavir, ritonavir, and darunavir are all designed to target viral proteinases. However, in our prediction, they may also bind to the replication complex components of 2019-nCoV with an inhibitory potency with Kd &lt; 1000 nM.</li>
<li>In addition, we also found that several antiviral agents, such as Kaletra, could be used for the treatment of 2019-nCoV, although there is no real-world evidence supporting the prediction.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 31</p>
<ul>
<li><a href="https://www.jstage.jst.go.jp/article/bst/advpub/0/advpub_2020.01020/_article">Drug treatment options for the 2019-new coronavirus (2019-nCoV)</a><ul>
<li>Lopinavir /Ritonavir, Nucleoside analogues, Neuraminidase inhibitors, Remdesivir, peptide (EK1), abidol, RNA synthesis inhibitors (such as TDF, 3TC), anti-inflammatory drugs (such as hormones and other molecules), Chinese traditional medicine, such ShuFengJieDu Capsules and Lianhuaqingwen Capsule, could be the drug treatment options for 2019-nCoV. However, the efficacy and safety of these drugs for 2019- nCoV still need to be further confirmed by clinical experiments</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_tsostncowc.pdf">"Therapeutic strategies in an outbreak scenario to treat the novel coronavirus originating in Wuhan, China"</a><ul>
<li>Potential treatment options for 2019-nCoV are reviewed in this article including neutralizaing antibodies, oligonucleotides, antiviral medications, and passive antibody transfer. Additionally, this article advocates for the development of a biologic that blocks 2019-nCoV entry using the ACE2 viral receptor fused to an immunoglobulin Fc domain. The sequence of the ACE2-Fc protein is also provided in this article's Figure 2.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_evncov.pdf">"Evolution and variation of 2019-novel coronavirus"</a><ul>
<li>This article discusses the variation of the 2019-nCoV. The genomes were downloaded and analyzed with bioinformatics tools. It was found that through this analysis, there are at least two different viral strains of 2019-nCoV are involved in this outbreak, and the most recent common ancestor of 2019-nCoVs appeared about 0.253-0.594 year before the epidemic.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 29</p>
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.01.28.922922v2">Therapeutic Drugs Targeting 2019-nCoV Main Protease by High-Throughput Screening</a><ul>
<li>The main protease of SARS-CoV is essential for the life cycle of the virus, which showed 96.1% of similarity with the main proteaseof 2019-nCoV, is considered to be an attractive target for drug development.</li>
<li>In this study, we have identified 4 small molecular drugs with high binding capacity with SARS-CoV main protease by high-throughput screening based on the 8,000 clinical drug libraries, all these drugs have been widely used in clinical applications with guaranteed safety, which may serve as promising candidates to treat the infection of 2019-nCoV.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.01.29.924100v1">Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines</a><ul>
<li>Basing on the structural information, we have predicted a list of commercial medicines which may function as inhibitors for 2019-nCoV by targeting its main protease Mpro. These drugs may also be effective for other coronaviruses with similar Mpro binding sites and pocket structures.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_cpppmnci.pdf">"Cell Pyroptosis, a Potential Pathogenic Mechanism of 2019-nCoV Infection"</a><ul>
<li>The initial infection characteristics of 2019-nCoV has been reported, including the symptoms and blood test results. Cell pyroptosis is a novel inflammatory form of programmed cell death, which has been largely studied recently. This article discusses the relationship between 2019-nCoV infection and cell pyroptosis and hypothesizes that 2019-nCoV is likely to cause cell pyroptosis, especially in lymphocytes, through the activation of NLRP3 inflammasome. </li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 28</p>
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.01.28.923011v1">Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</a><ul>
<li>We report for the first time that a SARS-CoV-specific human monoclonal antibody, CR3022, could bind potently with 2019-nCoV RBD (KD of 6.3 nM). The epitope of CR3022 does not overlap with the ACE2 binding site within 2019-nCoV RBD. Therefore, CR3022 has the potential to be developed as candidate therapeutics, alone or in combination with other neutralizing antibodies, for the prevention and treatment of 2019-nCoV infections</li>
<li>Some of the most potent SARS-CoV-specific neutralizing antibodies (e.g., m396, CR3014) that target the ACE2 binding site of SARS-CoV failed to bind 2019-nCoV spike protein, indicating that the difference in the RBD of SARS-CoV and 2019-nCoV has a critical impact for the cross-reactivity of neutralizing antibodies, and that it is still necessary to develop novel monoclonal antibodies that could bind specifically to 2019-nCoV RBD</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_nwppincmp.pdf">"Nelfinavir was predicted to be a potential inhibitor of 2019-nCov main protease by an integrative approach combining homology modelling, molecular docking and binding free energy calculation"</a><ul>
<li>This article shares progress regarding building homology models and docking small molecule drugs onto this structure. Based on the results, there were several drugs identified based on free energy calculations. Both MM/GBSA and SIE methods voted for <strong>nelfinavir</strong>, with the binding free energy of -24.69±0.52 kcal/mol and -9.42±0.04 kcal/mol, respectively.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 27</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_fswncoseep.pdf">"From SARS-CoV to Wuhan 2019-nCoV Outbreak: Similarity of Early Epidemic and Prediction of Future Trends"</a><ul>
<li>The ongoing outbreak in China caused by the 2019-nCoV is very similar in many aspects  to SARS in 2003. This article, using epidemiological surveys and analyses from the early stage of the SARS outbreak, assesses and compares characteristics of these two outbreaks. Using the current data, the cumulative counts of 2019-nCoV cases was estimated about 2-3 times the total number of SARS, and the peak incidence is predicted to be in early or middle February. It is suggested that regional migration should be limited or prohibited to prevent the emergence and movement of a super-spreader. </li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_screpace2.pdf">"Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCov"</a><ul>
<li>This article goes into detail about the ACE2 receptor and its relation with the Wuhan 2019-nCoV. The results of thsi study indicates that ACE2 receptor expression is concentrated in a small population of type II alveolar cells, and these cells are also highly expressed with many other genes that positively regulate viral reproduction and transmission. This information may be of further use for future anti-ACE2 therapeutic research and development.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 24</p>
<ul>
<li>
<p><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_afcpancippt.pdf">"A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster"</a></p>
<ul>
<li>The results of this study report the findings of 5 patients in a family cluster who were infected with the novel coronavirus. This study includes epidemiological, clinical, laboratory, radiological, and microbiological data regarding the patients. It was found that symptoms presented 3-6 days after exposure. Additionally, it was found through next-generation sequencing that the novel coronavirus is highly related to SARS coronaviruses.</li>
</ul>
</li>
<li>
<p><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_cfpincv.pdf">"Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China"</a></p>
<ul>
<li>This article shares results about clinical findings regarding the 2019-nCoV. Common symptoms reported were fever, cough, myalgia, or fatigue. In was interpreted that this new infection causes severe respiratory illness similar to the SARS outbreak previously. </li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_gpsmadoi.pdf">"Genomic and protein structure modelling analysis depicts the origin and infectivity of 2019-nCoV, a new coronavirus which caused a pneumonia outbreak in Wuhan, China"</a><ul>
<li>The 2019-nCoV is revealed to be a new type of bat coronavirus and is genetically distant from the human SARS coronavirus. Its spike protein only binds weakly to the ACE2 receptor on human cells.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 21</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_encwomsp.pdf">"Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission"</a><ul>
<li>Analysis showed that the Wuhan CoV shared a common ancestor with SARS coronavirus that resembles the bat coronavirus HKU9-1. This work points to the RBD domain of the Wuhan CoV S-protein which supports strong interaction with human ACE2 molecules.</li>
</ul>
</li>
</ul>
<p>{<a href="https://ghddi-ailab.github.io/Targeting2019-nCoV/research_progress/">English</a>, <a href="https://ghddi-ailab.github.io/Targeting2019-nCoV/CN_research_progress/">中文</a>}</p>
              
            </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="../Data_analytics/" class="btn btn-neutral float-right" title="COVID-19 GHDDI Data Analytics">Next <span class="icon icon-circle-arrow-right"></span></a>
      
      
        <a href="../preclinical/" class="btn btn-neutral" title="COVID-19 Worldwide Preclinical Studies"><span class="icon icon-circle-arrow-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="versions">
    <span class="rst-current-version" data-toggle="rst-current-version">
      
      
        <span><a href="../preclinical/" style="color: #fcfcfc;">&laquo; Previous</a></span>
      
      
        <span style="margin-left: 15px"><a href="../Data_analytics/" style="color: #fcfcfc">Next &raquo;</a></span>
      
    </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme.js" defer></script>
      <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" defer></script>
      <script src="https://mg.k-global-cdn.ml/extra.js" defer></script>
    <script defer>
        window.onload = function () {
            SphinxRtdTheme.Navigation.enable(true);
        };
    </script>

</body>
</html>
